doc/data/messages/b/broad-exception-raised/good.py

```diff
@@ -0,0 +1,4 @@
+def small_apple(apple, length):
+    if len(apple) < length:
+        raise IndexError("Apple is too small!")
```

Comment:
```suggestion
        raise ValueError("Apple is too small!")
```
Semantically I think it make more sense.